modules/AOW_WorkFlow/AOW_WorkFlow.php

```diff
@@ -457,10 +457,11 @@ function check_valid_bean(SugarBean &$bean){
 
         if($this->flow_run_on){
 
-            // database time correction with the user's time-zoneqq
-            $beanDateEnteredTimestamp = strtotime($timedate->asUser(new DateTime($timedate->fromDb($bean->date_entered))));
-            $beanDateModifiedTimestamp = strtotime($timedate->asUser(new DateTime($timedate->fromDb($bean->date_modified))));
-            $thisDateEnteredTimestamp = strtotime($this->date_entered);
+            // database time correction with user's time-zone (using user's date format)
+            $userDateFormat = $timedate->get_date_time_format();
+            $beanDateEnteredTimestamp = date_create_from_format($userDateFormat,$timedate->asUser(new DateTime($timedate->fromDb($bean->date_entered))))->getTimestamp();
```

Comment:
getTimestamp will cause a fatal if the datetime is null see the unit test results